installer/lib/mix/tasks/phx.new.ex

```diff
@@ -12,9 +12,12 @@ defmodule Mix.Tasks.Phx.New do
 
   ## Options
 
-    * `--live` - include Phoenix LiveView to make it easier than ever
+    * `--live` - include Phoenix.LiveView to make it easier than ever
       to build interactive, real-time applications
 
+    * `--dashboard` - include Phoenix.LiveDashboard which provides
+      real-time performance monitoring and debugging tools for developers.
+
```

Comment:
I guess this is redundant because it is default. Should we say anything about this other than the --no-dashboard below?

Reply:
Yup, remove this one!